graphql/util.go

```diff
@@ -709,6 +709,19 @@ func ModifyVersion(version model.Version, user user.DBUser, proj *model.ProjectR
 		if err := model.SetVersionPriority(version.Id, modifications.Priority, user.Id); err != nil {
 			return http.StatusInternalServerError, errors.Errorf("error setting version priority: %s", err)
 		}
+		// update priority for child patches
+		p, err := patch.FindOneId(version.Id)
+		if err != nil {
+			return http.StatusInternalServerError, errors.Wrapf(err, "error getting patch '%s'", version.Id)
+		}
+		if p == nil {
```

Comment:
This isn't necessarily an error, since ModifyVersion can be called for mainline commits too

Reply:
I modeled it after this in ModifyVersion. Is that incorrect ?
https://github.com/evergreen-ci/evergreen/blob/main/graphql/util.go#L675

Reply:
That block specifically checks that the requester is a merge test requester, i.e. we know it's a commit queue patch.

Reply:
true. thanks! fixed.